recipes/dsnparse/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "dsnparse" %}
+{% set version = "0.1.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6252d2a0b12e7c5bf1775e77ed0d3ee0a06f92937cdb2d2cac6ee039b557d0b4
+  patches:
+    - 0001-Include-License-in-source-distribution.patch
+
+build:
+  number: 0
```

Comment:
It looks like this could be `noarch: python`

https://conda-forge.org/docs/meta.html#building-noarch-packages